app/src/main/java/se/kth/sda/othello/imp/OthelloImp.java

```diff
@@ -688,10 +704,18 @@ private void swapPlayer() {
      * @author petrych
      */
     public void moveInitialNodes() {
```

Comment:
It seems like that number of initial nodes is redundant. You used them for testing right? Then we don't need them in the working code.
I think you can edit this method in your local branch, make a commit and push again, then we can merge to master.

Reply:
Oh, I guess I guess I forgot to change it back. 
I'll fix that first thing in the morning.